### ISSUE TITLE
auto page generator

### ISSUE THREAD

**Issue by Telomore** [janboone/open_source_for_economists#12]
In the instructions of "Set up Git" it contains the old layout of the settings page of github.
It is no longer possible to use the  "Launch automatic page generator".
github pages has changed: https://pages.github.com/   , it now faces restrictions.
With kind regards

**Comment by anuragbudakoti** [comment]
in my account launch automatic page generator command is not shown in github pages

**Comment by anuragbudakoti** [comment]
![Uploading 20190907_150357.jpg…]()

**Comment by anuragbudakoti** [comment]
![20190907_150357](https://user-images.githubusercontent.com/55012516/64472885-b8ecef80-d182-11e9-8e56-edc52dd4d484.jpg)

**Comment by janboone** [comment]
Indeed, this is confusing as github changed the interface. I will take this part offline till I have time to correct is.

On your screenshot, you see source. Click on the arrow next to "None" and select the branch that you want to use for your github pages.
